index.html

```diff
@@ -1,11 +1,467 @@
+                src="./images/services--1.jpg"
+                alt="First image in services"
+                width="370"
+              />
+            </li>
+
+            <li class="services__item service">
+              <img
+                class="service__image"
+                src="./images/services--2.jpg"
+                alt="Second image in services"
+                width="370"
+              />
+            </li>
+
+            <li class="services__item service">
+              <img
+                class="service__image"
+                src="./images/services--3.jpg"
+                alt="Third image in services"
+                width="370"
+              />
+            </li>
+          </ul>
+        </div>
+      </section>
+
+      <section class="section team">
+        <div class="container">
+          <h2 class="team__title section__title">Наша команда</h2>
+          <ul class="team__list">
+            <li class="team__item person">
+              <img
+                class="person__image"
+                src="./images/team--1.jpg"
+                alt="Igor Demyanenko"
+                width="270"
+              />
+              <div class="person__info">
+                <h3 class="person__name">Ігор Дем'яненко</h3>
+                <p class="person__role">Product Designer</p>
+
+                <ul class="person__social social">
+                  <li class="social__item">
+                    <a class="social__link" href="#instagram">
+                      <svg class="social__icon">
+                        <use href="./images/sprite.svg#instagram"></use>
+                      </svg>
+                    </a>
+                  </li>
+
+                  <li class="social__item">
+                    <a class="social__link" href="#twitter">
+                      <svg class="social__icon">
+                        <use href="./images/sprite.svg#twitter"></use>
+                      </svg>
+                    </a>
+                  </li>
+
+                  <li class="social__item">
+                    <a class="social__link" href="#facebook">
+                      <svg class="social__icon">
+                        <use href="./images/sprite.svg#facebook"></use>
+                      </svg>
+                    </a>
+                  </li>
+
+                  <li class="social__item">
+                    <a class="social__link" href="#linkedin">
+                      <svg class="social__icon">
+                        <use href="./images/sprite.svg#linkedin"></use>
+                      </svg>
+                    </a>
+                  </li>
+                </ul>
+              </div>
+            </li>
+
+            <li class="team__item person">
+              <img
+                class="person__image"
+                src="./images/team--2.jpg"
+                alt="Olga Repina"
+                width="270"
+              />
+              <div class="person__info">
```

Comment:
![image](https://github.com/user-attachments/assets/e610dc83-07e4-4426-adba-3ad0d0b16ec7)

now it would make sense to add horizontal paddings to the **`person__info`** element to limit the **`social`** block width